src/TMap.cpp

```diff
@@ -1741,44 +1741,6 @@ bool TMap::retrieveMapFileStats( QString profile, QString * latestFileName = 0,
         if( areaCount ) {
             * areaCount = areaSize;
         }
-        // read each area
```

Comment:
As far as I see, ``pA`` is filled in and never read from again. Could you show me where it is needed?